include/bm/bm_sim/calculations.h

```diff
@@ -387,6 +387,32 @@ class CustomCrcMgr {
                                           const crc_config_t &config);
 };
 
+enum class ToeplitzRssErrorCode {
+  SUCCESS = 0,
+  INVALID_CALCULATION_NAME,
+  WRONG_TYPE_CALCULATION,
+  INVALID_KEY,
+};
+
+namespace detail {
+
+using toeplitz_rss_key_t = std::vector<uint8_t>;
+
+std::ostream &operator<<(std::ostream &out, const toeplitz_rss_key_t &rss_key);
+
+}  // namespace detail
+
+class ToeplitzRssMgr {
+ public:
+  using toeplitz_rss_key_t = detail::toeplitz_rss_key_t;
```

Comment:
since this is in the scope of `ToeplitzRssMgr`, you could change it to `using rss_key_t = detail::toeplitz_rss_key_t;` if you want